etc/jenkins/build_linux.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#
+# CI build script for building couchbase-lite-java{-ee} on linux platforms.
+#
+
+LITE_CORE_REPO_URL="http://nexus.build.couchbase.com:8081/nexus/content/repositories/releases/com/couchbase/litecore"
+MAVEN_URL="http://172.23.121.218/maven2/cimaven"
+
+function usage() {
+    echo "Usage: $0 <build number> <edition, CE or EE>"
+    exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+BUILD_NUMBER="$1"
+if [ -z "$BUILD_NUMBER" ]; then
+    usage
+fi
+
+EDITION="$2"
+if [ -z "$EDITION" ]; then
+    usage
+fi
+
+touch local.properties
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
really minor point:  "SCRIPT_DIR" seems like an odd name, given that $CBL_JAVA_DIR/scripts is where all the scripts are...

Reply:
It automatically determine where the script is located so the script can be run from any location.

Reply:
Yeah!  It works fine... I just thought the name was a little odd, given that the scripts that you run (which might be said to be in a scripts directory) were somewhere else!  No worries.  It's fine.